### ISSUE TITLE
Process Env no longer inherited by tasks

### ISSUE THREAD

**Issue by spmeesseman** [Microsoft/vscode#71826]
Tested with insiders, bug still present.

- VSCode Version: 1.33.0
- OS Version: Windows 10

Steps to Reproduce:

1.  Install 1.33.0.  Process environment is no longer inherited by shell tasks (defined in tasks.json).
2.  Unzip 1.32.3 over the 1.33 install, environment once again is inherited by shell tasks.
3.  Unzip 1.34 Insiders over the install, problem returns.
4. Launch with --disable-extensions, problem remains.
4.  Put 1.32.3 back, problem gone.

If this isnt a bug what changed that I need to do now?  I combed the release notes but found nothing.

Does this issue occur when all extensions are disabled?: No

**Comment by spmeesseman** [comment]
Just a follow up note.... A team of 14 here, no one can update because this problem is affecting ALL of our builds.  We spent days and days making VSCode task files when we switched to VSCode, now none of them work in this version since there is no environment inherited.

**Comment by spali** [comment]
can reproduce this exactly the same way with the integrated terminal in general.
using something like this in the settings
```
  "terminal.integrated.env.windows": {
    "Path": "${env:Path}",
  }
```
wipes out the whole path variable because ${env:Path} is always empty.
before 1.33 it works as expected.

**Comment by davidwin** [comment]
Looks like #72029

**Comment by spmeesseman** [comment]
fixed in 1.33 Recovery 1
